web/src/routes/admin/settings/+page.svelte

```diff
@@ -0,0 +1,239 @@
+
+      // Add a pause if it is scheduled after the current timeslot
+      if (breakIndex < breaks.length && breaks[breakIndex].afterSlot === i) {
+        const breakLength = breaks[breakIndex].length;
+        const breakEndTime = addMinutesToDate(currentStartTime, breaks[breakIndex].length);
+
+        timeslotList.push({
+          from: formatDateToTimeString(currentStartTime),
+          to: formatDateToTimeString(breakEndTime),
+          relativeIndex: breakIndex,
+          type: Type.PAUSE,
+        });
+
+        currentStartTime = breakEndTime;
+        breakIndex++;
+      }
+    }
+
+    return timeslotList;
+  }
+
+  async function saveMetadata() {
+    await setTimetableMetadata(metaData);
+    toast.success('Erfolgreich', {
+      description: 'Die Einstellungen wurden erfolgreich gespeichert!',
+    });
+  }
+
+  function convertTimeStringToDate(timeString: string): Date {
+    const [hours, minutes] = timeString.split(':').map(Number);
+    const date = new Date();
+    date.setHours(hours);
+    date.setMinutes(minutes);
+    return date;
+  }
+
+  function addMinutesToDate(date: Date, minutes: number): Date {
+    const newDate = new Date(date);
+    newDate.setMinutes(newDate.getMinutes() + minutes);
+    return newDate;
+  }
+
+  function formatDateToTimeString(date: Date): string {
+    const hours = date.getHours();
+    const minutes = date.getMinutes();
+
+    const formattedHours = hours.toString().padStart(2, '0');
+    const formattedMinutes = minutes.toString().padStart(2, '0');
+
+    return `${formattedHours}:${formattedMinutes}`;
+  }
+
+  function updateTimeslots(event) {
+    const value = parseInt(event.target.value, 10);
+    if (value !== metaData.timeslotsAmount) {
+      changed = true;
+      metaData.timeslotsAmount = value;
+      metaData.breaks = metaData.breaks.filter((b) => b.afterSlot < metaData.timeslots);
+    }
+  }
+
+  function updateStartTime(event) {
+    if (event.target.value !== metaData.startTime) {
+      changed = true;
+      metaData.startTime = event.target.value;
+    }
+  }
+
+  function updateTimeSlotLength(event) {
+    if (event.target.value !== metaData.timeslotLength) {
+      metaData.timeslotLength = parseInt(event.target.value);
+      changed = true;
+    }
+  }
+
+  function hasTimeslotBreak(index: number): boolean {
+    for (let i = 0; i < metaData.breaks.length; i++) {
+      const break1 = metaData.breaks[i];
+      if (break1.afterSlot === index) {
+        return true;
+      }
+    }
+    return false;
+  }
+</script>
+
+<div class="flex flex-row justify-start bg-foreground md:p-4 text-white">
+  <Button
+    on:click={async () => {
+      await goto('/admin');
+    }}
+    variant="secondary"
+    size="icon"
+    class="rounded-full bg-accent mr-6"
+  >
+    <ChevronLeft class="h-5 w-5 text-white" />
+  </Button>
+  <h1 class="font-bold text-xl mt-1">{$page.data['meta']['title']}</h1>
+</div>
+
+{#await getTimetableMetadata() then metadata}
```

Comment:
Dont you call `getTimetableMetadata()` in the load function, isn't this await not needed?